### PR TITLE
Revised and moved the fixed statement article

### PR DESCRIPTION
**Pull request by pkulikov**

### CODE REVIEW

File: docs/csharp/language-reference/statements/fixed.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# fixed statement (C# reference)
+
+The `fixed` statement prevents the [garbage collector](../../../standard/garbage-collection/index.md) from relocating a moveable variable and declares a pointer to that variable:
+
+:::code language="csharp" source="snippets/fixed/Program.cs" id="PinnedArray":::
+
+The address of a fixed, or pinned, variable doesn't change for the duration of the statement. You can use the declared pointer only inside the corresponding `fixed` statement. The declared pointer is readonly and cannot be modified.
+
+> [!NOTE]
+> You can use the `fixed` statement only in an [unsafe](../keywords/unsafe.md) context. The code that contains unsafe blocks must be compiled with the [**AllowUnsafeBlocks**](../compiler-options/language.md#allowunsafeblocks) compiler option.
```
Comment (pkulikov) [inline docs/csharp/language-reference/statements/fixed.md:21]: @BillWagner question not for this PR, but for possible upcoming work: what do you think about the article about `unsafe`:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/unsafe

Now, it's in TOC under "Keywords -> Modifiers" as `unsafe` may be used as a type/method modifier. Do you want the dedicated article about `unsafe` statement under "Statements" in TOC? If yes, then that article would be very short and I guess the current article under modifiers still should stay. What do you think?

Comment (BillWagner) [reply docs/csharp/language-reference/statements/fixed.md:21]: Hi @pkulikov 

In time, I'd like to have one area of the **Language Reference** section devoted to *unsafe code* including `fixed`, `unsafe`, pointer types, and so on.  My reasoning is that the recent language enhancements around `ref` (including the `ref` fields and `scoped` additions) mean that it's much less likely for developers to need any unsafe constructs. I'd like to put those constructs together in one section.

**Review by BillWagner — APPROVED** [review]
This all LGTM @pkulikov 

I'll :shipit: now.

File: docs/csharp/language-reference/statements/fixed.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/statements/fixed.md:21]: @BillWagner ok, clear, thanks. Then, is it already fine in meantime to "merge" the `unsafe` keyword article into the [Unsafe code article](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/unsafe-code) as one more section in it?

Comment (BillWagner) [reply docs/csharp/language-reference/statements/fixed.md:21]: @pkulikov 

> is it already fine in meantime to "merge" the unsafe keyword article into the [Unsafe code article](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/unsafe-code) as one more section in it?

Yes. Thanks for asking.

**Pull request merged**